crates/biome_cli/src/commands/mod.rs

```diff
@@ -77,11 +77,24 @@ pub enum BiomeCommand {
     #[bpaf(command)]
     Check {
         /// Apply safe fixes, formatting and import sorting
-        #[bpaf(long("apply"), switch)]
+        #[bpaf(long("apply"), switch, hide_usage)]
         apply: bool,
         /// Apply safe fixes and unsafe fixes, formatting and import sorting
-        #[bpaf(long("apply-unsafe"), switch)]
+        #[bpaf(long("apply-unsafe"), switch, hide_usage)]
```

Comment:
note; Hide `--apply`, `--apply-unsafe`, please check snapshot diff for the help command.